easybuild/easyconfigs/d/DIAMOND/DIAMOND-0.9.36-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'CMakeMake'
+
+name = 'DIAMOND'
+version = "0.9.36"
+
+homepage = 'https://github.com/bbuchfink/diamond'
+description = "Accelerated BLAST compatible local sequence aligner"
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
```

Comment:
@mboisson Same as in #11683: isn't GCC a better choice here, since this is performance sensitive?